src/reducers/org.js

```diff
@@ -1312,10 +1317,10 @@ function updatePlanningItemsWithRepeaters({
     currentTodoSet.get('keywords').first()
   );
   if (!noLogRepeatEnabledP({ state, headerIndex })) {
-    const lastRepeatTimestamp = timestampForDate(new Date(), {
+    const lastRepeatTimestamp = timestampForDate(timestamp, {
```

Comment:
I was stuck at this point... what is the parameter `currentDate`? Isn't it also a `new Date()` timestamp, like your new parameter `timestamp`? Why was it not used in this place? I coudn't find out yet.

Reply:
I'm sorry, I might not fully understand the question. I'll try to answer: Yes, in this spot, there was a call to `new Date()` as it was in others. I replaced them all with the same `new Date()` call made in an action: https://github.com/200ok-ch/organice/pull/423/commits/cf6e2849c5fd40e73b6203e69a9401c4bef54c8a

Reply:
I mean, this function (line 1306) already has a parameter `currentDate`. Now it has `timestamp` and `currentDate` – are they the same thing?

Reply:
Ah, now I get it. Thank you for persisting^^

It's a good catch. `currentDate` is dead code which isn't even passed from the action - alongside other dead code:

![image](https://user-images.githubusercontent.com/505721/91657352-97d28c80-eac0-11ea-937b-313bc914f1e2.png)

Reply:
This even leads to buggy behaviour of code thinking the variables are defined:

![image](https://user-images.githubusercontent.com/505721/91657420-03b4f500-eac1-11ea-8200-5366086bafe6.png)

I'll clean this up.